docs/customization/fixtures.rst

```diff
@@ -25,29 +25,35 @@ the ``sylius_fixtures.fixture`` tag in order to be used in suite configuration.
     The former interface extends the ``ConfigurationInterface``, which is widely known from ``Configuration`` classes
     placed under ``DependencyInjection`` directory in Symfony bundles.
 
-Why would you customize a fixtures?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Why would you customize fixtures?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you want to operate on the test data of objects appearing in your store
-e.g: ``Currency``, ``Country``, ``User`` etc.
+There are two main use cases for customizing fixture suites, in each of them you can adapt the data of your shop to be realistic,
+the default fixtures suite of Sylius is selling clothes, if you are selling food you'd probably need your own fixtures to show that:
 
-How to add new Fixtures?
-~~~~~~~~~~~~~~~~~~~~~~~~
+    * preparing test data for the development purposes like demo applications prepared for QA
+    * preparing the shop configuration for the production instance
 
-In Sylius part of fixtures are added in ``src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures.yml``, where is defined ``default`` suite that is partially-configured.
-If you are planning to add new fixtures in default command ``sylius:fixtures:load``, only what you have to do is add new items in ``config\packages\_sylius.yml``
+How to modify the existing Sylius fixtures?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Adding new items in shop by standard yaml configuration:
+In Sylius, fixtures are added in ``src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures.yml``.
+We have the ``default`` suite that is partially-configured.
+If you are planning to modify the default fixtures applied by the ``sylius:fixtures:load`` command, the only thing that you have
```

Comment:
If you are planning to modify the default fixtures applied by the ``sylius:fixtures:load`` command, modify the ``config\packages\sylius_fixtures.yaml`` file.